.github/title-labeler.yml

```diff
@@ -0,0 +1,20 @@
+version: v1
+
+labels:
+  - label: 'bug-fix'
+    sync: false
+    matcher:
+      # no case-insensitive flag (https://github.com/fuxingloh/multi-labeler/issues/341)
+      title: '.*(fix|Fix|bug|Bug|correctly|Correctly|use correct|Use correct).*'
+  - label: 'chore'
+    sync: false
+    matcher:
+      title: '.*(upgrade|Upgrade).*'
+  - label: 'enhancement'
+    sync: false
+    matcher:
+      title: '.*(improve|Improve|enhance|Enhance|refine|Refine).*'
+  - label: 'feature'
+    sync: false
+    matcher:
+      title: '.*(add support|Add support).*'
```

Comment:
Why do we care about the distinction here? Is there one?

- We don't have a `feature` label (well, I guess we do now that the bot created one for this PR).
- The "enhancement" label description is "A desired new feature or change (not a bug)" (already seems to encompass new features).
- Your [new issue templates](https://github.com/PrairieLearn/PrairieLearn/pull/11629) treat them synonymously too.

Reply:
I am going crazy, I must have thought that was one of ours last night.